### DISCUSSION TITLE
webchat icon

### DISCUSSION THREAD

**Discussion by jusce17** [botfront/botfront#934] [Q&A] [answered]
Hey guys, I am pretty new using botfront (just started now)

I am using the rasa-webchat and it is working perfectly. however I would like to see how I can change the icon of the webchat.

I would like to use my bot's avatar instead of the conversation icon that shows up there.

Thanks

**Comment by jusce17** [comment, marked as answer]
For those who might face the same questions, all you need it to pass the path to a param named profileAvatar


```
  WebChat.default.init({

    selector: "#webchat",

    customData: {"language": "en"}, // arbitrary custom data. Stay minimal as this will be added to the socket

     profileAvatar: "url to your photo"

  })
```

**Comment by rahulDatahive** [comment]
Hi , i'm also new to botfront  ,do you know how to open the webchat by default and not on click?
